runtime/src/accounts.rs

```diff
@@ -256,13 +255,13 @@ impl AccountsDB {
     pub fn transaction_count(&self) -> u64 {
         self.transaction_count
     }
-    pub fn account_values_slow(&self) -> Vec<(Pubkey, solana_sdk::account::Account)> {
-        self.accounts.iter().map(|(x, y)| (*x, y.clone())).collect()
-    }
-    fn merge(&mut self, other: Self) {
-        self.transaction_count += other.transaction_count;
-        self.accounts.extend(other.accounts)
-    }
+    //pub fn account_values_slow(&self) -> Vec<(Pubkey, solana_sdk::account::Account)> {
```

Comment:
@aeyakovenko, https://github.com/solana-labs/solana/blob/master/CONTRIBUTING.md#should-i-send-small-prs-as-i-develop-large-new-components